scripts/re-id/baseline/process/transforms.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import absolute_import
+import os, numbers, random, math
+
+from mxnet.gluon.data import dataset
+from mxnet import image, nd
+from mxnet.gluon import Block
+
+
+class Pad(Block):
```

Comment:
transform can be moved to https://github.com/dmlc/gluon-cv/blob/master/gluoncv/data/transforms/block.py